patientsearch/config.py

```diff
@@ -25,7 +25,12 @@ def load_json_config(potential_json_string):
 ENABLE_PATIENT_DELETE = os.getenv("ENABLE_PATIENT_DELETE", "false").lower() == "true"
 
 ENABLE_PROVIDER_FILTER = os.getenv("ENABLE_PROVIDER_FILTER", "false").lower() == "true"
-
+FILTER_FOR_TEST_PATIENTS_LABEL = os.getenv(
+    "FILTER_FOR_TEST_PATIENTS_LABEL", "Include test patients"
```

Comment:
I updated just now on isacc-dev, but think this config is overkill, seems that we've settled on a good value and as we know, less config, is more ;)